### ISSUE TITLE
[Adaptive Cards - iOS - Action.ShowCard.Json]: After activating the 'Dismiss' button, voice over focus is not landing on the 'Neat' button.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards-Mobile#165]
### Target Platforms

iOS

### SDK Version

Version 1.0 (2.4.0-beta.24.5.30.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 17.5.1
Application: Adaptive Cards
App Version: Version 1.0 (2.4.0-beta.24.5.30.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.0" control and select it.
5. Navigate to 'Elements' control and activate it.
6. Navigate to 'Action.ShowCard.Json' and activate it.
7. Card will appear.
8. Perform right swipe and navigate to the 'Neat' button and double tap to activate it.
9. Perform right swipe and navigate to the dismiss button and double tap to activate it and observe the issue.

**Actual Result:**
Voice over focus is not retain back on the Neat button when user activates Dismiss button. After activating the dismiss button voice over focus is landing on the top of the page.

**Expected Result:**
Voice over focus should be retained on the same control when user activates dismiss button. After activating dismiss button, voice over focus should be land on the Neat button.

**User Impact:**
It will impact the screen reader user as they need to put more effort to renavigate on the same control due to this extra time will also spent.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/47eb59a3-01be-4730-882e-ceb37072d2fe

### Card JSON

```JSON
NA
```
